src/components/Home.js

```diff
@@ -1,10 +1,51 @@
+/* eslint-disable */
+// noinspection ES6UnusedImports,JSUnusedLocalSymbols
+
+import { useEffect, useState} from 'react';
+import { useTheme } from '../App';
 import { TeacherPage } from './TeacherPage';
 import { StudentsPage } from './StudentsPage';
+import { SnowFlakes } from './SnowFlakes';
+import { DelayedRedirect } from './DelayedRedirect';
+import * as Api from '../Api';
+import { ReactComponent as TreeIcon } from '../images/tree.svg';
+import '../styles/Login.css';
+
+const Home = ({ token, darkMode, setDarkMode, snowFlakes, setSnowFlakes }) => {
+    const [userType, setUserType] = useState(undefined);
+
+    if (token === undefined) {
+        // return (
+        //     <div className={ getClassNameByTheme("login-redirect") }>
+        //         <TreeIcon />
+        //         <h1>You have to log in to use this website!</h1><br/>
+        //         <h1>Redirecting...</h1>
+        //         <DelayedRedirect to="/login" delay={ 3500 }/>
+        //     </div>
+        // )
+    }
+
+    // eslint-disable-next-line
+    useEffect(() => {
+        const fetchUserType = async () => {
+            // setUserType(await Api.getUserType(token));  // TODO: enable this after development
+            setUserType('teacher');  // TODO: remove this after development
+        };
+
+        // noinspection JSIgnoredPromiseFromCall
+        fetchUserType();
+        // eslint-disable-next-line
+    }, []);
 
-const Home = () => {
     return (
         <div>
+            { userType === 'student' && <StudentsPage token={ token } darkMode={ darkMode } setDarkMode={ setDarkMode }
+                                                      snowFlakes={ snowFlakes } setSnowFlakes={ setSnowFlakes } /> }
+            { userType === 'teacher' && <TeacherPage darkMode={ darkMode } setDarkMode={ setDarkMode }
+                                                     snowFlakes={ snowFlakes } setSnowFlakes={ setSnowFlakes } /> }
+            { userType === undefined && <div>Hello There!</div> /* TODO: handle this error */ }
```

Comment:
Add error handling here.

Reply:
It's in TODO

Reply:
Complete all TODOs before merging.

Reply:
That's impossible

Reply:
Than don't open a pull request. If you complete all the TODOs I commented on here, that is enough for a merge.